TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/test_cases/multi_type_tests.py

```diff
@@ -91,9 +91,9 @@ def perform_operations_after_backfill_migration(self):
 
     def perform_operations_during_live_capture_migration(self):
         self.target_operations.get_document(cluster=self.target_cluster, index_name=self.index_name,
-                                            doc_id=self.doc_id3, max_attempts=10, delay=3.0)
+                                            doc_id=self.doc_id3, max_attempts=20, delay=3.0)
         self.target_operations.get_document(cluster=self.target_cluster, index_name=self.index_name,
-                                            doc_id=self.doc_id4, max_attempts=10, delay=3.0)
+                                            doc_id=self.doc_id4, max_attempts=20, delay=3.0)
```

Comment:
Why do we have a pattern of so many retries, shouldn't the workflow be deterministic enough that each step should reasonable complete on the first attempt?

Reply:
Until we do, can we have a default pre-backfill complete action that monitors for the backfill to be done, or better yet something like that in the console, `console backfill wait-until-complete` or whatever the cleanest pattern is?

Reply:
I created this task to track work for reporting a COMPLETED status for backfill, which we can then easily check in integration tests to wait for a completed state: https://opensearch.atlassian.net/browse/MIGRATIONS-2449